src/gui/README.md

```diff
@@ -345,6 +345,19 @@ To remove all the rulers:
 gui::clear_rulers
 ```
 
+### Heat Maps
```

Comment:
The change to underlining in the 'Display Control' for indicating a double-click dialog wasn't obvious.   Please add a description of that to the REAME

Reply:
I added some language about it, but at some point it might be nice to fill out the README with pictures to help explain what the GUI is showing and how to use it.